contrib/devtools/gen-manpages.sh

```diff
@@ -11,7 +11,9 @@ ZCASHTX=${ZCASHTX:-$SRCDIR/zcash-tx}
 [ ! -x $ZCASHD ] && echo "$ZCASHD not found or not executable." && exit 1
 
 # The autodetected version git tag can screw up manpage output a little bit
-ZECVER=($($ZCASHCLI --version | head -n1 | awk -F'[ -]' '{ print $5, $6 }'))
+ZECVERSTR=$($ZCASHCLI --version | head -n1 | awk '{ print $NF }')
+ZECVER=$(echo $ZECVERSTR | awk -F- '{ OFS="-"; NF--; print $0; }')
+ZECCOMMIT=$(echo $ZECVERSTR | awk -F- '{ print $NF }')
```

Comment:
Phew. Took a bit of reading awk and interactively running these with `v1.0.8-1` to know what this means. :-o